frontend/src/app/sessions/session/session.component.html

```diff
@@ -99,7 +102,12 @@
       <div [ngClass]="draggingActive ? 'iframe-overlay' : ''"></div>
       <iframe
         [id]="'session-' + session.id"
-        class="w-full height"
+        [ngClass]="{
+          'w-full': true,
+          'heigh-fullscreen': fullscreenService.isFullscreen$ | async,
+          height: (fullscreenService.isFullscreen$ | async) === false
+        }"
+        class="w-full height max-w-full max-h-full"
```

Comment:
```suggestion
        class="w-full"
```

If the `width` / `height` is set to a fixed value, there shouldn't be the need of setting a max value. Height should be handled in the parent container (which contains the session title bar and the iframe). To pass the height down, set `h-full` on the `iframe-container` element.

Reply:
I completely reworked the styling and tried to remove as much unnecessary css as possible. I think its best to just have another look over it.